tests/unit/notice_validator/cli/test_cmd_validation_summary_runner.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from ted_sws.notice_validator.entrypoints.cli.cmd_validation_summary_runner import main as cli_main, \
+    DEFAULT_OUTPUT_PATH, DEFAULT_TEST_SUITE_REPORT_FOLDER
+
+
+def post_process_for_notice(fake_repository_path, fake_mapping_suite_id, fake_validation_notice_id):
+    base_path = fake_repository_path / fake_mapping_suite_id / DEFAULT_OUTPUT_PATH
+    notice_report_path = base_path / fake_validation_notice_id / DEFAULT_TEST_SUITE_REPORT_FOLDER
+    assert os.path.isdir(notice_report_path)
```

Comment:
I recommend to use pathlib instead of os for Paths.

Reply:
It's not the case here. They are similar for this case